docs/getting-started/installation.md

```diff
@@ -16,6 +16,10 @@ $ curl -LsSf https://astral.sh/uv/install.sh | sh
 $ powershell -c "irm https://astral.sh/uv/install.ps1 | iex"
 ```
 
+!!! tip
+
+    For PowerShell 7.X replace `powershell` with `pwsh`. [See Microsoft Docs for differences](https://learn.microsoft.com/en-us/powershell/scripting/whats-new/differences-from-windows-powershell)
```

Comment:
iirc this would also apply to 6.x

Reply:
Ok. Update to 6/7.X or just say PowerShell Core?

Reply:
Do more Windows users think of it in terms of Windows PowerShell vs. PowerShell Core, or 5.x vs 6/7.x?

Reply:
At least in my experience I do say `PowerShell Core` over 6.x/7.x, but unsure about the community.